README.md

```
@@ -20,6 +20,10 @@ Read the documentation on [docs.source.network](https://docs.source.network/).
 
 <!--ts-->
    * [Install](#install)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)